File: Ticket_booking_API/Ticket_booking_API/Models/Booking.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Ticket_booking_API.Models
{
    public class Booking
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public int TicketId { get; set; }
        public Ticket Ticket { get; set; }
        public int Count { get; set; }


    }
}

File: src/app/booking.ts
import { AnimationKeyframesSequenceMetadata } from "@angular/animations";

export class Booking {
    id:number;
    userId:number;
    user:any;
    ticketId:number;
    ticket:any;
    count:number;
    constructor()
    {
        this.id=0;
        this.userId=0;
        this.user=null;
        this.ticketId=0;
        this.ticket=null;
        this.count=0;
        
    }
}
